### DISCUSSION TITLE
How can I use gradient in a border?

### DISCUSSION THREAD

**Discussion by lucasviga** [chakra-ui/panda#2296] [Q&A]
Hey guys... I want to create a border gradient with 3 colors but I don't have any idea how to do it.

In the documentation, I just found about bgGradient but not border gradient.

You guys have any idea how to create this style?

Note: my project is in Next.JS 14.

**Comment by anubra266** [comment]
@lucasviga gradient border is not a feature in css

**Reply by lucasviga** [reply]
yeah, I know that. 

I was looking for way to set a gradient in a border, border-image might help it. Thanks, anyway.
